source/tutorials/openvino.rst

```diff
@@ -0,0 +1,249 @@
+
+#. To view the OpenVINO Sample Application source code, run the command:
+
+   .. code-block:: console
+
+      ls /usr/share/doc/inference_engine/samples
+
+In the next section, you learn how to use an OpenVINO Sample Application.
+
+Run OpenVINO Sample Application
+*******************************
+
+After installing OpenVINO on |CL|, you need a model against which to test.
+In this example, we use the public squeezenet 1.1 model for image
+classification.
+
+Use a model to test
+===================
+
+#. If you don’t have any model, you can download an
+   **intel_model** or a public model using OpenVINVO Model Downloader.
+
+   - Check the list of public models you can download from
+     :file:`/usr/share/open_model_zoo/models/public`
+
+   - Check the list of Intel® models you can download from
+     :file:`/usr/share/open_model_zoo/intel_models`
+
+#. View the location of OpenVINO Model Downloader:
+
+   .. code-block:: console
+
+      cd /usr/share/open_model_zoo/tools/downloader
+
+#. In general, download models with the following command:
+
+   .. code-block:: bash
+
+      python3 downloader.py --name <model_name> -o <downloading_path>
+
+   .. note::
+
+      * Where :file:`<model_name>` is the one you chose from **intel_model**
+        above
+
+      * Where :file:`<downloading_path>` is your project directory
+
+#. For this example, run:
+
+   .. code-block:: bash
+
+      python3 downloader.py --name squeezenet1.1 -o $HOME/.
+
+#. After running this command, the model appears as downloading at your
+   :file:`$HOME/classification/squeezenet/1.1/caffe` as follows:
+
+   .. code-block:: console
+
+      ###############|| Downloading topologies ||###############
+
+      ========= Downloading /$HOME/classification/squeezenet/1.1/caffe/squeezenet1.1.caffemodel
+      ... 100%, 4834 KB, 2839 KB/s, 1 seconds passed
+
+      ...
+
+Convert model to IR format
+==========================
+
+#. As necessary, follow the instruction on :ref:`convert-dl-models`
+   to convert deep learning models.
+
+#. Navigate to the model:
+
+   .. code-block:: bash
+
+	  cd $HOME/classification/squeezenet/1.1/caffe
+
+#. Run the command:
+
+   .. code-block:: bash
+
+      python3 /usr/share/openvino/model-optimizer/mo.py --input_model squeezenet1.1.caffemodel
+
+
+   The output will show these files being generated:
+
+   .. code-block:: console
+
+      squeezenet1.1.xml
+
+      squeezenet1.1.bin
+
+#. Finally, run :command:`ls` to view the newly added model and files.
+
+Run image classification
+========================
+
+This sample application demonstrates how to run the Image Classification in asynchronous mode on supported devices, using Classification_sample_async.
+
+.. TODO: Command needs review; it failed. See email. Need: Change directory?
+
+#. In general, to execute the sample application, run following command:
+
+   .. code-block:: bash
+
+      classification_sample_async -i <path_to_image> -m <path_to_model_ir> -d <device>
```

Comment:
Running the above command, using the image of an automobile, was not successful. Will you please advise? See also my email.